### DISCUSSION TITLE
v1.2.4

### DISCUSSION THREAD

**Discussion by TitanNano** [TitanNano/TS4ControlAnySim#60] [Releases]
## What's Changed
* Version 1.2.3 was accidentally marked as being a preview version when it was built. This new release corrects this mistake, but otherwise does not differ from the previous version.

**Full Changelog**: https://github.com/TitanNano/TS4ControlAnySim/compare/v1.2.3...v1.2.4

<hr /><em>This discussion was created from the release <a href='https://github.com/TitanNano/TS4ControlAnySim/releases/tag/v1.2.4'>v1.2.4</a>.</em>

**Comment by ladynimue** [comment]
Are there any plans to add control for "Stay Over Guests"?  This is a new feature since the Growing Together pack and and Control Any Sim only adds the sim's grayed out icon to the panel but doesn't let you select them.  Thank you for your time and consideration.  Love your mod.
